config/config.toml.example

```diff
@@ -25,8 +25,8 @@ user = "sparky"
 role = "pipeline"
 password = "sparky123"
 dbname = "pipeline"
-logging = true
-autoMigrateEnabled = false
+logging = false # Turn this to true to see all queries in the log
+autoMigrateEnabled = true # This should be false in production
```

Comment:
It should be false by default

Reply:
It is false by default, this is just an example configuration file which is used to generate a development configuration. Locally it should be true IMHO.

Reply:
i see, thanks